lib/transactions.js

```diff
@@ -83,12 +93,60 @@ export const renderDetailsString = ({
     intl.locale,
   );
 
-  const detailString = [amountString, hostFeeString, platformFeeString, paymentProcessorFeeString];
-  if (taxAmount) {
-    detailString.push(formatFee(taxAmount, amount, taxInfo?.id || 'Taxes', taxInfo?.percentage, intl.locale));
-  }
+  if (kind === TransactionKind.EXPENSE) {
+    const expensePaymentProcessorFee = formatCurrency(
+      Math.abs(paymentProcessorFee.valueInCents),
+      paymentProcessorFee.currency,
+      { locale: intl.locale },
+    );
+    const netExpenseAmount = formatCurrency(Math.abs(_netAmount.valueInCents), amount.currency, {
+      locale: intl.locale,
+    });
+    const payee = toAccount.name;
+    const payer = fromAccount.name;
+    const feesPayer = expense.feesPayer;
 
-  return detailString.concat(' ');
+    return (
+      <React.Fragment>
+        <Container mb={1}>
+          {paymentProcessorFee.valueInCents !== 0 ? (
```

Comment:
Why not display that all the time?

Reply:
So my reasoning here is that if there's no payment processor fee; 

1) It can be that there's no payment processor fee in the transaction. In this case the net amount and expense amount are equal so I thought there's no point of displaying any of this info. So what we see is;

![image](https://user-images.githubusercontent.com/12435965/153640866-c806de9a-0bb9-4855-b251-8a2c83b22b78.png)

This is consistent with what we see for added funds. I was debating in my mind as to whether remove the expense details section for this case altogether but decided to keep it for now.

2) For a refund transaction where there's no refunded payment processor fee. For refund of a transaction where payee pays payment processor fee and the payment processor fee is not refunded. For this case the collective cover the payment processor fee but at least for now we don't display that detail here. 

![image](https://user-images.githubusercontent.com/12435965/153641705-34a578d8-56fe-4a8c-a09a-64c088cb8ec4.png)

Also note that at https://github.com/opencollective/opencollective/issues/5220 I am trying to display the payment processor cover and that PR will add the `Refunded Payment Processor Fee` label to all of the refunds (except 2 above). So this section will change there anyways. 🤔

Reply:
I'm not comfortable having exceptions and having a different output depending on the case. Users will ask, "why is it not displayed like the others?". We need consistency.

Reply:
I understand. So here we could display as same as before, `Payment Processor Fees: 0` and I'll do that if no objections. The reason I didn't do this was that somehow I thought it wasn't nice to show `0` (or $0.00) there. But I think we can iterate later if needed on this. 🤔 🤔

Reply:
Done. 👍🏽 

![image](https://user-images.githubusercontent.com/12435965/153862569-4ac61569-c73c-45c0-82d4-a0f14c1901d3.png)